src/worker.ts

```diff
@@ -20,7 +21,10 @@ export default {
 	async scheduled(req: ScheduledController, env: env, ctx: ExecutionContext) {
 		switch (req.cron) {
 			case EVERY_6_HOURS: {
-				return await callDiscordNicknameBatchUpdate(env);
+				await callDiscordNicknameBatchUpdate(env);
+				await filterOrphanTasks(env);
+				console.log('Worker for filtering the orphan tasks has completed');
```

Comment:
await? won't this cause workers to exhaust 30s timelimit

Reply:
Sir, without `await` how we can call the async function, I mean yes you are right the case is real where we might be exhausting the 30 time-limit. But I then how to do it without `await`